spec/mongoid/association/embedded/embeds_many/proxy_spec.rb

```diff
@@ -1426,12 +1453,186 @@ class TrackingIdValidationHistory
       it "returns true" do
         expect(person.addresses.any?).to be true
       end
+
+      it "block form iterates across all documents" do
+        expect(person.addresses.any? {|a| a.street == "Upper" }).to be false
+        expect(person.addresses.any? {|a| a.street == "Bond" }).to be true
+      end
+
+      context 'ruby version >= 2.5' do
+        ruby_version_gte '2.5'
+
+        it "argument form is supported" do
+          expect(person.addresses.any?(person.addresses.first)).to be true
+          expect(person.addresses.any?(1)).to be false
+        end
+      end
     end
 
-    context "when documents are not created" do
+    context "when documents are not present" do
       it "returns false" do
         expect(person.addresses.any?).to be false
       end
+
+      it "block form iterates across all documents" do
+        expect(person.addresses.any?(&:a)).to be false
+      end
+
+      context 'ruby version >= 2.5' do
+        ruby_version_gte '2.5'
+
+        it "argument form is supported" do
+          expect(person.addresses.any?(1)).to be false
+        end
+      end
+    end
+  end
+
+  describe "#all?" do
+
+    let(:person) do
+      Person.create
```

Comment:
These should all use `create!` to verify the creation succeeded.

Reply:
Done. I've changed all usages of .create in the spec file.